doc/en/components/charts/chart-type-overview.md

```diff
@@ -1,32 +1,212 @@
 _keywords: $PlatformShort$ Charts, Chart, Infragistics
 ---
-# $PlatformShort$ Charts
-
-<!-- TODO add introduction to charts -->
-
-## Types of $PlatformShort$ Charts
-
-- [Area Chart](chart-types-area.md)
-- [Bar Chart](chart-types-bar.md)
-- [Bubble Chart](chart-types-scatter.md#$PlatformShort$-Scatter-Bubble-Chart)
-- [Column Chart](chart-types-column.md)
-- [Donut Chart](../doughnut-chart.md)
-- [Financial Chart](chart-types-financial.md)
-- [Line Chart](chart-types-line.md)
-- [Polar Chart](chart-types-polar.md)
-- [Pie Chart](../pie-chart.md)
-- [Radial Chart](chart-types-radial.md)
-- [Scatter Chart](chart-types-scatter.md)
-- [Shape Chart](chart-types-scatter.md#$PlatformShort$-Scatter-Polygon-Chart)
-- [Stacked Chart](chart-types-area.md#$PlatformShort$-Stacked-Area-Chart)
-- [Spline Chart](chart-types-spline.md)
-- [Step Chart](chart-types-step.md)
-- [Sparkline](../sparkline.md)
-- [Treemap](../treemap-overview.md)
-
-## List of Chart Features
-
-<!-- tooltips, legends, axes, overlays, annotations, etc. -->
\ No newline at end of file
+# $PlatformShort$ Charts & Graphs Overview
+
+Ignite UI for $PlatformShort$ Charts & Graphs is an extensive library of data visualizations that enable stunning, interactive charts and dashboards for your web and mobile apps. Built for speed and beauty, designed to work on every modern browser, and with complete touch and interactivity, you can quickly and easily build responsive visuals into your next app on any device. 
+
+The $PlatformShort$ charting component supports over 65 types of series and combinations that let you visualize any type of data, including Category Series, Financial Series, Polar Series, Radial Series, Range Series, Scatter Series, Shape Series, and Geospatial Series. No matter the type of comparison you are doing, or what type of data story you are trying to tell, you can represent your data in any of these ways: 
+
+- Change Over Time 
+- Comparison 
+- Correlation 
+- Distribution 
+- Geospatial 
+- Overview + Detail 
+- Part to Whole 
+- Ranking 
+
+Power your most demanding visualizations with Infragistics $PlatformShort$ charting! 
+
+## $PlatformShort$ Chart Example
+
+This $PlatformShort$ column chart, or column graph, example shows Olympics medals by Country. The $PlatformShort$ column chart belongs to a group of category charts that include at one or more numeric or time-series based data point, and one string data point, that compares data values by differing column heights. In this example, the Y-Axis, or left-side labels of the chart, are displaying numeric data showing the number of medals won, and the X-Axis, or bottom labels, are showing a time value, in this case Year. The columns are the category values representing different countries, like USA, Japan and Germany. 
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
```

Comment:
@StefanIvanov Should we have this example with a more complex chart or a simpler one? Like this one for example:

https://share.goabstract.com/02743744-824f-4699-954e-8f001e608fc0?collectionLayerId=cd37a0da-0b42-4e6f-9014-f383c22bdfd2&mode=design

Reply:
I think this is a good sample for here, bar charts are one of the most popular chart types and it is good to continue from here onward with such a sample for a beginning.

Reply:
We will use this example in the Column chart section and I will design something different here. I will try and find a data that combines charts like columns and line for instance.

Reply:
I found the data and based on it I suggest we use this example:

https://share.goabstract.com/749407bb-319b-4ab5-a280-900e1afbcd29?mode=build&sha=febbb137bfbe8ef0751fa55b2239f2326e2adc71

Reply:
Love this sample

Reply:
Here is the data:

[facebook Downloadable-PL-Q4-2020.xlsx](https://github.com/IgniteUI/igniteui-xplat-docs/files/5912357/facebook.Downloadable-PL-Q4-2020.xlsx)

Reply:
@HUSSAR-mtrela As far as I know we are combining Category and Data charts, and there will no longer be any devision between them, so we can use whichever example we decide. I think this is a good sample combining two/three types of charts, showcasing a lot of our capabilities.

Reply:
@HUSSAR-mtrela I believe we may use such a Data Chart e.g. columns with trendlines and area in one of the scenario-related topics if @jberes creates such a scenario. I'd keep the overview more focused on simpler samples of key chart features to show how easy it is to use.